Gifski/Base.lproj/MainMenu.xib

```diff
@@ -66,6 +66,12 @@
                                     <action selector="open:" target="-1" id="E6Y-20-4aZ"/>
                                 </connections>
                             </menuItem>
+                            <menuItem title="Quick Look..." keyEquivalent=" " id="Wfh-bM-KvX">
```

Comment:
```suggestion
                            <menuItem title="Quick Look" keyEquivalent=" " id="Wfh-bM-KvX">
```

Reply:
And I think there should be a separator above this item.

Reply:
Yeah, was thinking about that as well. Gonna add it and see.